README.md

```diff
@@ -149,7 +149,7 @@ TopoJSON always consists of a single topology object. A topology may contain any
 
 A topology is a TopoJSON object where the type member's value is “Topology”.
 
-  * A topology must have a member with the name “objects” whose value is another object, each of which is a named geometry object.
+  * A topology must have a member with the name “objects” whose value is another object, whose each property is a geometry object.
```

Comment:
Okay, I pushed some new wording. Let me know if that’s better or send another PR with further edits. Thanks!